apps/cancel-request/translations/src/en/fields.json

```diff
@@ -125,6 +125,46 @@
         "hint": "You are acting on behalf of a child named as a dependant on the application"
       }
     }
+  },
+  "cnc-reference-number": {
+    "legend": "Which reference number can you provide?",
+    "options": {
+      "record-number": {
+        "label": "Record number",
+        "hint": "This is in the confirmation email you received when you submitted. For example, ROD123456789"
+      },
+      "case-id": {
+        "label": "Case ID",
+        "hint": "The main applicant received this number when they applied. For example, 12345678"
+      },
+      "ho-reference-number": {
+        "label": "Home Office reference number",
+        "hint": "This is on official documents, such as letters about your immigration status or biometric residence permit cards. For example, A1234567"
+      },
+      "payment-reference-number": {
+        "label": "Payment reference number",
+        "hint": "This is on the letter the Home Office sent to the main applicant confirming they received an application"
+      },
+      "courier-reference-number": {
+        "label": "Courier reference number",
+        "hint": "This includes Royal Mail reference numbers"
+      }
+    }
+  },
+  "enter-record-number": {
+    "label": "Enter a record number"
+  },
+  "enter-case-id": {
+    "label": "Enter the case ID"
+  },
+  "enter-ho-reference-number": {
+    "label": "Enter the Home Office reference number"
+  },
+  "enter-payment-reference-number": {
+    "label": "Enter a payment reference number"
+  },
+  "enter-courier-reference-number": {
+    "label": "Enter a courier reference number"
```

Comment:
The labels for these classes aren't consistent, but then the figma only really shows 'Enter the record number' I think probably they should all say 'Enter the x number' but can double check with ucd to make sure?

Reply:
Yes, figma shoes only for the record number so I put rest of them based on the error messages. But yes, good point. I will double check this

Reply:
Sent a note to ucd to confirm on this.